### PR TITLE
move outboard moment and shear equations to aircraft models

### PR DESCRIPTION
**Pull request by mjburton11**

### CODE REVIEW

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/wing/sparloading.py [hunk 1]
```diff
@@ -65,22 +62,19 @@ def setup(self, wing, state, sp=False):
...
         deta = self.wing.planform.deta
 
-        if sp:
-            with SignomialsEnabled():
-                shear = S[:-1] >= (S[1:] + 0.5*deta*(b/2.)*(q[:-1] + q[1:])
-                                   - Sout)
-        else:
-            shear = S[:-1] >= S[1:] + 0.5*deta*(b/2.)*(q[:-1] + q[1:])
+        constraints = []
+        if not sp:
```
Comment (bqpd) [inline gpkitmodels/GP/aircraft/wing/sparloading.py]: the `sp` argument appears to be unused now?

Comment (mjburton11) [reply gpkitmodels/GP/aircraft/wing/sparloading.py]: Well, in the solar SP model I can't have those constraints appear so... that why the `sp` arg is still there.

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/motor/motor.py [hunk 2]
```diff
@@ -41,7 +41,7 @@ class Motor(Model):
 
     Variables
     ---------
-    Qstar       .8          [kg/(N*m)]         motor specific torque
+    Qstar       .5          [kg/(N*m)]         motor specific torque
```
Comment (courtin) [inline gpkitmodels/GP/aircraft/motor/motor.py]: .5 is pretty sporty based on the numbers from the STOL guys; .8 is more in line with their model at representative powers

Comment (mjburton11) [reply gpkitmodels/GP/aircraft/motor/motor.py]: It wasn't solving for higher values.  I can try again.  Also not convinced that the STOL guys numbers are that accurate...

**Comment by mjburton11** [comment]
test models please

File: gpkitmodels/GP/aircraft/motor/motor.py (hunk 2 above)
Comment (courtin) [reply gpkitmodels/GP/aircraft/motor/motor.py]: Yea it was a fit of that one manufacturer so you may be able to get lighter; it's a high-sensitivity variable for sure.

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
test this please

**Comment by mjburton11** [comment]
test models please

**Comment by mjburton11** [comment]
@galbramc @bqpd looks like macys couldn't install ad for some reason

**Comment by mjburton11** [comment]
test this please

**Comment by galbramc** [comment]
I don't know why pip randomly fails to install things. I can't see a consistent pattern...

**Pull request merged**